credentials/prc-vc-v2/credential.json

```
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://w3id.org/citizenship/v4rc1"
+  ],
+  "id": "https://issuer.oidp.uscis.gov/credentials/83627465",
```

Use generic issuer URL.
```suggestion
  "id": "https://issuer.example.gov/credentials/83627465",
```